File: Examples/CSharp/ProgrammersGuide/WorkingWithProjects/ReadFiscalYearProperties.cs
using System.IO;
using Aspose.Tasks;
using System;

namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects
{
    public class ReadFiscalYearProperties
    {
        public static void Run()
        {
            //ExStart: ReadFiscalYearProperties
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Projects();
            //Create a project reader instance
            Project project = new Project(dataDir+ "ReadFiscalYearProperties.mpp");

            //Display fiscal year properties
            Console.WriteLine("Fiscal Year Start Date : " + project.Get(Prj.FyStartDate).ToString());
            Console.WriteLine("Fiscal Year Numbering : " + project.Get(Prj.FiscalYearStart).ToString());
            //ExEnd: ReadFiscalYearProperties
        }
    }
}

File: Examples/VisualBasic/ProgrammersGuide/WorkingWithProjects/ReadFiscalYearProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports System

Namespace Aspose.Tasks.Examples.VisualBasic.WorkingWithProjects
    Public Class ReadFiscalYearProperties
        Public Shared Sub Run()
            'ExStart: ReadFiscalYearProperties
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Projects()
            'Create a project reader instance
            Dim project As New Project(dataDir & "ReadFiscalYearProperties.mpp")

            'Display fiscal year properties
            Console.WriteLine("Fiscal Year Start Date : " & project.Get(Prj.FyStartDate).ToString())
            Console.WriteLine("Fiscal Year Numbering : " & project.Get(Prj.FiscalYearStart).ToString())
            'ExEnd: ReadFiscalYearProperties
        End Sub
    End Class
End Namespace